### ISSUE TITLE
pkg-config stuff is missing for libopenzwave1.3-dev package

### ISSUE THREAD

**Issue by pastukhov** [OpenZWave/open-zwave#687]
Seems like libopenzwave.pc generation is wrong and missing in libopenzwave1.3-dev package creation spec.

**Comment by Fishwaldo** [comment]
Wrong? In what way. Can you provide a bit more detail than a one liner please?

**Comment by pastukhov** [comment]
I'm trying to build packages for Debian jessie directly on my Banana Pi:
```dpkg-buildpackage -b```
* libopenzwave.pc file is missing from libopenzwave1.3-dev_0.9_armhf.deb.
I found it in ./debian/tmp/usr/local/lib/arm-linux-gnueabihf/pkgconfig/libopenzwave.pc:
```
prefix=/usr
exec_prefix=/usr/bin
libdir=/usr/lib/arm-linux-gnueabihf/
includedir=/usr/include/openzwave/
sysconfdir=/usr/etc/openzwave/
gitversion=0.9.-1
docdir=/usr/share/doc/openzwave-0.9.0/

Name: libopenzwave
Description: A Open Source implementation of the ZWave Serial API
URL: http://www.openzwave.org/
Version: 0.9
Libs: -L${libdir} -lopenzwave
Cflags: -I${includedir} 
```
* So it has been generated in wrong place and having wrong version number.

**Comment by Fishwaldo** [comment]
Could you please post the output of "pkg-config --variable pc_path pkg-config"

**Comment by Fishwaldo** [comment]
Pending on a fix from PR #689

**Comment by jjhuff** [comment]
Curious if anyone has a solution for this. I'm using the Debian stretch package, and it's missing libopenzwave.pc (https://packages.debian.org/stretch/amd64/libopenzwave1.5-dev/filelist)

Thanks!
